docs/Contributing/Issues/issues.md

```diff
@@ -0,0 +1,70 @@
+## Using the issue tracker
+
+The [issue tracker](https://github.com/RI-SE/ATOS/issues) on the repo page is the preferred channel for [bug reports](#bugs) and features requests. If you want to submit a pull request that relates to a specific issue or bug, please select the bug in question and use the "create branch for this issue or link a pull request" option under the debelopment heading to link your pull request to the issue.
+
+![image](https://github.com/RI-SE/ATOS/assets/10436623/adb9d512-ab25-4504-aa9d-63c95ba76c85)
+
+
+In general, also please respect the following restrictions:
+
+* Please **do not** use the issue tracker for personal support requests. [The discussions tab](https://github.com/RI-SE/ATOS/discussions) on the repo can be used for this instead. 
+
+* Please **do not** derail or troll issues. Keep the discussion on topic and
+  respect the opinions of others.
+
+
+<a name="bugs"></a>
+## Bug reports
+
+A bug is an error, flaw or fault in the design of a piece of software or code.
+We welcome bug reports, as they are extremely helpful in furthering the development of the project.
+
+Guidelines for bug reports:
+
+1. **Use the GitHub issue search** &mdash; check if the issue has already been
+   reported.
+
+2. **Check if the issue has been fixed** &mdash; try to reproduce it using the
+   latest `main` or development branch in the repository.
+
+3. **Isolate the problem** &mdash;
+A good bug report shouldn't leave others needing to chase you up for more
+information. Please try to be as detailed as possible in your report. What is
+your environment? What steps will reproduce the issue? What OS are you using, which
+browser is used for the gui?What would you expect to be the outcome? All these
+details will help people to fix any potential bugs.
+
+The issue report template is present in the issues tab on gihub, but the main points are also shown below:
+
+## Issue Template
```

Comment:
This says Issue template, but below it says "Describe the bug", confusing.

Is this the issue template or the bug template?